# Type: test
# Subtype: T1
# English Summary: Delete test cases: optionalSensitiveTest and optionalNoDefaultTest
# Chinese Summary: 删除测试用例：optionalSensitiveTest和optionalNoDefaultTest
# Entry Number: 2

# === Git Diff Content ===

diff --git a/syncany-lib/src/test/java/org/syncany/tests/plugins/PluginOptionSpecTest.java b/syncany-lib/src/test/java/org/syncany/tests/plugins/PluginOptionSpecTest.java
index 1026396f..e5dbb5f5 100644
--- a/syncany-lib/src/test/java/org/syncany/tests/plugins/PluginOptionSpecTest.java
+++ b/syncany-lib/src/test/java/org/syncany/tests/plugins/PluginOptionSpecTest.java
@@ -31,16 +31,6 @@ import org.syncany.plugins.PluginOptionSpec.ValueType;
  */
 public class PluginOptionSpecTest {
 	
-	@Test(expected=IllegalArgumentException.class) 
-	public void optionalSensitiveTest() {
-		new PluginOptionSpec("id", "desc", ValueType.STRING, false, true, null);		
-	}
-	
-	@Test(expected=IllegalArgumentException.class) 
-	public void optionalNoDefaultTest() {
-		new PluginOptionSpec("id", "desc", ValueType.STRING, false, false, null);		
-	}
-	
 	@Test
 	public void nonSetMandatoryTest() {
 		PluginOptionSpec pluginOptionSpec = new PluginOptionSpec("id", "desc", ValueType.STRING, true, true, null);
